tools/scoring/README.md

```diff
@@ -0,0 +1,13 @@
+# Scoring
+
+The scoring tool evaluates _proposed_ configuration files against "known good" _solution_ files. It uses an F-score algorithm to provide floating-point representations of how closely the files compare along various _dimensions_.
+
+## Install
+To install the dependencies, please run the `python3 setup.py install` from the following directories, in order:
+* digitalbuildings/tools/validators/ontology_validator
+* digitalbuildings/tools/validators/instance_validator
+* digitalbuildings/tools/scoring
+
+## Usage
+
+From the `scoring` directory: `python3 score/cli.py -ont path/to/ontology/yaml/resources -prop path/to/proposed/file.yaml -sol path/to/solution/file.yaml -v True`
```

Comment:
can we please rename this `cli.py` to `score.py` just to be compliant with the other tools on github

Reply:
I tried to implement the naming scheme you suggested but kept encountering issues with module resolution (due to naming conflicts, I believe). I will revisit.